modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -163,6 +163,19 @@ def get_most_recent_visited_clinic_appointment
         nil
       end
 
+      def get_recent_sorted_clinic_appointments
+        end_time = Date.current.end_of_day.yesterday
+        start_time = 1.year.ago
+        statuses = 'booked,fulfilled,arrived'
+
+        appointments = fetch_clinic_appointments(start_time, end_time, statuses)
```

Comment:
Since I was going off of the logic from the previous endpoint I had only considered kind=clinic but I could see those other types of appts potentially being of interest, would be curious to hear @simiadebowale thoughts when he gets a chance to take a look

Reply:
I think we should retrieve clinic info for all appointment types

Reply:
Noted, will update to make the general `get_appointments` call as opposed to the selective `fetch_clinic_appointments` call